From randy.dunlap@oracle.com  Tue Feb 17 17:40:35 2009
From: Randy Dunlap <randy.dunlap@oracle.com>
Date: Fri, 13 Feb 2009 11:22:06 -0800
Subject: USB: fix ehci printk formats
To: Stephen Rothwell <sfr@canb.auug.org.au>
Cc: Greg KH <greg@kroah.com>, Andrew Morton <akpm@linux-foundation.org>
Message-ID: <4995C85E.9040001@oracle.com>


From: Randy Dunlap <randy.dunlap@oracle.com>

Fix ehci printk formats:

drivers/usb/host/ehci-q.c:351: warning: format '%d' expects type 'int', but argument 4 has type 'size_t'
drivers/usb/host/ehci-q.c:351: warning: format '%d' expects type 'int', but argument 5 has type 'size_t'

Signed-off-by: Randy Dunlap <randy.dunlap@oracle.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/host/ehci-q.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/usb/host/ehci-q.c
+++ b/drivers/usb/host/ehci-q.c
@@ -349,7 +349,7 @@ qh_completions (struct ehci_hcd *ehci, s
 						--qh->xacterrs > 0 &&
 						!urb->unlinked) {
 					ehci_dbg(ehci,
-	"detected XactErr len %d/%d retry %d\n",
+	"detected XactErr len %zu/%zu retry %d\n",
 	qtd->length - QTD_LENGTH(token), qtd->length,
 	QH_XACTERR_MAX - qh->xacterrs);
 
